sktime/transformers/summarise.py

```diff
@@ -215,3 +228,286 @@ def transform(self, X, y=None):
+        self.n_jobs = n_jobs
+        self.show_warnings = show_warnings
+        self.disable_progressbar = disable_progressbar
+        self.profile = profile
+        self.profiling_filename = profiling_filename
+        self.profiling_sorting = profiling_sorting
+        self.test_for_binary_target_binary_feature = test_for_binary_target_binary_feature
+        self.test_for_binary_target_real_feature = test_for_binary_target_real_feature
+        self.test_for_real_target_binary_feature = test_for_real_target_binary_feature
+        self.test_for_real_target_real_feature = test_for_real_target_real_feature
+        self.fdr_level = fdr_level
+        self.hypotheses_independent = hypotheses_independent
+        self.ml_task = ml_task
+
+
+        # extractor and selector
+        self.feature_extractor = None
+        self.feature_selector = None
+    
+    def fit(self,X,y):
+
+        #input checks
+        validate_X(X)
+
+        # if self.timeseries_container is None:
+        #     raise RuntimeError("You have to provide a time series using the set_timeseries_container function before.")
+
+        self.feature_extractor = TsFreshFeatureExtractor(
+            default_fc_parameters=self.default_fc_parameters,
+            kind_to_fc_parameters=self.kind_to_fc_parameters,
+            chunksize=self.chunksize,
+            n_jobs=self.n_jobs,
+            show_warnings=self.show_warnings,
+            disable_progressbar=self.disable_progressbar,
+            profile=self.profile,
+            profiling_filename=self.profiling_filename,
+            profiling_sorting=self.profiling_sorting)
+
+
+        self.feature_selector = FeatureSelector(
+            test_for_binary_target_binary_feature=self.test_for_binary_target_binary_feature,
+            test_for_binary_target_real_feature=self.test_for_binary_target_real_feature,
+            test_for_real_target_binary_feature=self.test_for_real_target_binary_feature,
+            test_for_real_target_real_feature=self.test_for_real_target_real_feature,
+            fdr_level=self.fdr_level,
+            hypotheses_independent=self.hypotheses_independent,
+            n_jobs=self.n_jobs,
+            chunksize=self.chunksize,
+            ml_task=self.ml_task
+        )
+
+        # Check to include old features
+        X_old = X
+
+        #Transform timseries data
+        X_transformed = self.feature_extractor.transform(X_old)
+        print(X_transformed.head())
+
+        self.col_to_max, self.col_to_min, self.col_to_median = get_range_values_per_column(X_transformed)
+
+        #Immpute timeseries data
+        X_transformed = impute_dataframe_range(X_transformed, col_to_max=self.col_to_max, col_to_median=self.col_to_median,
+                                             col_to_min=self.col_to_min)
+
+        # print("Imputed X is",X_transformed.head())
+        # Fit using transformed timeseries data and y
+        self.feature_selector.fit(X_transformed, y)
```

Comment:
The sklearn convention is that fitted attributes have a trailing underscore, e.g.
```python
self.feature_selector_ = self.feature_selector.fit(X_transformed, y)
```